controllers/cloud.redhat.com/providers/cronjob/impl.go

```diff
@@ -41,6 +41,14 @@ func (j *cronjobProvider) makeCronJob(cronjob *crd.Job, app *crd.ClowdApp) error
 	if err := j.Cache.Update(CoreCronJob, c); err != nil {
 		return err
 	}
+
```

Comment:
This needs to go before the cache update, otherwise the changes will never get commited to the resource.

Reply:
updating, thx